spec/notifications/new_aircraft_created_notification_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe(NewAircraftCreatedNotification, type: :model) do
+  # Must eagerly create a user first, or notifications will not be created
+  before { create(:user) }
+
+  context "when aircraft are created" do
+    it "creates database notifications" do
+      expect { create(:aircraft) }.to(change(Notification, :count).by(1))
+    end
+
+    it "creates email notifications" do
+      expect { create(:aircraft) }.to(change { UserMailer.deliveries.count }.by(1))
+    end
+
+    it "sends the proper email subject" do
+      create(:aircraft)
+      email = ActionMailer::Base.deliveries.last
+      expect(email.subject).to(include("New Aircraft Created")) # replace by translated comment
```

Comment:
Can we support internationalization here? Is that what your comment means?
```suggestion
      expect(email.subject).to(include(I18n.t("notices.aircraft.updated"))) # replace by translated comment
```

Reply:
Yup, that's what my comment was getting at. Done!